index.html

```diff
@@ -291,23 +291,24 @@ <h1>Get Started</h1>
             </div>
             
             <div class="mobile-form">
-                <form action="submit">
+                <form action="https://formspree.io/f/myyldona" method="POST">
                     <div class="mobile-formcontainer">
                         <div class="mobile-fistname">
                             <label for="mobilefname">First Name</label>
                             <br>
-                            <input type="text" id="mobilefname" name="firstname" placeholder="Your name..">
+                            <input type="text" id="mobilefname" name="firstname" maxlength="30" required placeholder="Your name..">
                         </div>
                         <div class="mobile-email">
                             <label for="mobileemail">E-mail</label>
                             <br>               
-                            <input type="text" id="mobileemail" name="email" placeholder="Your e-mail..">
+                            <input type="email" id="mobileemail" name="_replyto" placeholder="Your e-mail..">
```

Comment:
- [ ] Email should include required attribute. Please fix this.

Reply:
Please, enlighten me.

Reply:
Sorry, my mistake.